lnwallet/channel.go

```diff
@@ -6723,6 +6736,24 @@ func (lc *LightningChannel) UpdateFee(feePerKw chainfee.SatPerKWeight) error {
 		EntryType: FeeUpdate,
 	}
 
+	// Make sure updating the fee won't violate any of the constraints we
```

Comment:
Potentially move this check into `validateFeeRate`?

Reply:
I found it most natural to do it hear, similar to other methods validating updates, and since we need the `paymentDescriptor`